miden-test-utils/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "miden-test-utils"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.67"
+
+[dependencies]
+assembly = { package = "miden-assembly", path = "../assembly", version = "0.5", default-features = false }
+processor = { package = "miden-processor", path = "../processor", version = "0.5", features = ["internals"], default-features = false }
+proptest = "1.1"
+prover = { package = "miden-prover", path = "../prover", version = "0.5", default-features = false }
+stdlib = { package = "miden-stdlib", path = "../stdlib", version = "0.4", default-features = false }
```

Comment:
Do we need `stdlib` dependency here? I think we might run into issues when we want to use it as dev dependency for tests in `stdlib`.